pom.xml

```diff
@@ -35,6 +36,8 @@
     </parent>
 
     <properties>
+        <maven.compiler.source>1.8</maven.compiler.source>
```

Comment:
These settings enforce the projects in this repository to build jars that are compatible with Java 8 by default. Some projects (bolt-google-cloud-functions, bolt-helidon in the short term) may overwrite the version.

Maintainers must use OpenJDK 11 (or newer stable LTS version in the future) for building jars from now on.